keps/sig-api-machinery/4052-generic-controlplane/kep.yaml

```diff
@@ -0,0 +1,19 @@
+name: generic-controlplane
+title: Add generic control plane staging repository
+kep-number: "4050"
```

Comment:
kep-number should be the number of an issue. I open an issue #4080 for this pr.

Reply:
ack